external/radiation/setup.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+from setuptools import setup, find_packages
+
+setup(
+    description="A python port of the GFS radiation scheme",
+    name="radiation",
+    author="The Allen Institute for Artificial Intelligence",
+    author_email="noahb@allenai.org",
+    license="BSD license",
+    version="0.1.0",
+    python_requires=">=3.8.0",
+    packages=find_packages(),
+    install_requires=["numba>=0.54.1", "xarray", "numpy", "netcdf4"],
```

Comment:
is numba already in `constraints.txt`? if not then you should run make lock_pip.

Reply:
It is already there, set at `==0.53.1`. The radiation nix env is using `0.54.1`. I was tempted to just check that the port runs OK at `0.53.1` and leave the constraints alone, but I guess I'll add the constraint from nix into the fv3net and prognostic run requirements.

Reply:
Oh. I think it's probably ok. One trick to enforce a constraint without explicitly adding it someplace is to set it in constraints.txt. `make lock_pip` will only change it if it needs to.